### PR TITLE
Continuous integration/deployment

### PR DESCRIPTION
**Pull request by adborden**
- Adds no-op for tests via `make test`
- Adds CI pipeline for test and deploy (deploys the master branch to "production")

### CODE REVIEW

File: www.foia.gov/README.md [hunk 1]
```diff
@@ -1,14 +1,37 @@
...
+at deploy time via CI/CD. Create the `.htpasswd` file at the root of the site
+directory:
 
-The Basic Auth functionality is present during pre-ATO process.
+    $ htpasswd -c -b .htpasswd theusername thepassword
 
-To create the .htpasswd file, use:
 
- % htpasswd -c -b .htpasswd theusername thepassword
+## Deployment
```
Comment (bsweger) [inline www.foia.gov/README.md]: Thanks for updating this and formatting it a bit--much appreciated!

**Review by bsweger — COMMENTED** [review]
Heya @adborden....looks good. The only thing I'm wondering about before we merge is: do we need to ignore the password file?

I was able to deploy this, so I think we're good on that front. 

Once this is merged, I'll take a stab at the local dev instructions and submit a PR if anything needs clarified. Thanks for your help!

File: www.foia.gov/README.md [hunk 2]
```diff
@@ -1,14 +1,37 @@
...
+the `foia-proxy.php` script, which points at the original site.
+
+
+## Authentication
+
+The Basic Auth functionality is present during pre-ATO process. Team members
+should request the username/passphrase from their teammates.
+
+The user and password is specified in the `.htpasswd` as a hash which is created
+at deploy time via CI/CD. Create the `.htpasswd` file at the root of the site
```
Comment (adborden) [inline www.foia.gov/README.md]: Oops yes. I'll add it to a `.gitignore`.

**Review by bsweger — APPROVED** [review]
Thanks for adding the `.gitignore`!

**Pull request merged**